build.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+################################################################################
+# Build spring boot with starter script
+# Usage:
+# bash build.sh [/path/to/installation/dir]
+################################################################################
+
+################################################################################
+# Define default values for variables
+################################################################################
+# no defaults yet!
+
+################################################################################
+# START DECLARATION FUNCTIONS
+################################################################################
+
+################################################################################
+function usage {
+################################################################################
+  echo Script for creating metastore service.
+  echo USAGE:
+  echo   $0 [/path/to/installation/dir]
+  echo IMPORTANT: Please enter an empty or new directory as installation directory.
+  exit 1
+}
+
+################################################################################
+function checkParameters {
+################################################################################
+  # Check no of parameters.
+  if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters"
+    usage
+  fi
+
+  # Check if argument is given
+  if [ -z "$1" ]; then
+    echo Please provide a directory where to install.
+    usage
+    exit 1
+  fi
+  
+  # Check for invalid flags
+  if [ "${1:0:1}" = "-" ]; then
+    usage
+  fi
+
+  INSTALLATION_DIRECTORY=$1
```

Comment:
As autocompletion may add a / as a suffix to the given folder name, should we remove it in this case?

```suggestion
  INSTALLATION_DIRECTORY=$1
  INSTALLATION_DIRECTORY=${INSTALLATION_DIRECTORY%"/"}
```

Reply:
Relative paths are always a problem. I transfer the installation directory to an absolute one. So this is no longer necessary.